### PR TITLE
Main

### PR DESCRIPTION
**Pull request by eihabkhan**

### CODE REVIEW

**Comment by jonniebigodes** [comment]
@eihabkhan quick question for you, why did you open a new pull request when you have the translation in #493? If this is related to this [comment](https://github.com/chromaui/learnstorybook.com/pull/493#issuecomment-859735281), just update it in your translation. No need to open a new pull request

**Comment by eihabkhan** [comment]
> @eihabkhan quick question for you, why did you open a new pull request when you have the translation in #493? If this is related to this [comment](https://github.com/chromaui/learnstorybook.com/pull/493#issuecomment-859735281), just update it in your translation. No need to open a new pull request

Yes, it was indeed related to that [comment](https://github.com/chromaui/learnstorybook.com/pull/493#issuecomment-859735281), sorry about that I didn't quite understand, but now I do, I will be using main for future translations, thank you @jonniebigodes, and my apologies for the inconvenience.

**Comment by jonniebigodes** [comment]
@eihabkhan no need to apologize whatsoever and don't worry about it. We're all learning here 🙂! As you've mentioned if this will be the pull request for the translation if you don't mind, close the other one and we'll take it from here. Sounds good?

**Comment by eihabkhan** [comment]
> @eihabkhan no need to apologize whatsoever and don't worry about it. We're all learning here 🙂! As you've mentioned if this will be the pull request for the translation if you don't mind, close the other one and we'll take it from here. Sounds good?

Sounds great! Understood.

**Comment by jonniebigodes** [comment]
@merodiro is there anything from your end that needs addressing in terms of proofreading? I'll do another sweep tomorrow (Wednesday) or Thursday at the latest and if nothing stands out I'll merge this afterward.

Let me know.

Stay safe

**Comment by merodiro** [comment]
I will proofread it tomorrow with the help of a friend I created a PR in the forked repo https://github.com/eihabkhan/learnstorybook.com/pull/1 I'm not sure if this is the right way

I believe the fixes that I made in the previous PR didn't make it in this PR. should I add them again?

**Comment by eihabkhan** [comment]
> I will proofread it tomorrow with the help of a friend I created a PR in the forked repo [eihabkhan#1](https://github.com/eihabkhan/learnstorybook.com/pull/1) I'm not sure if this is the right way
> 
> I believe the fixes that I made in the previous PR didn't make it in this PR. should I add them again?

I've reintegrated your fixes in [this](https://github.com/chromaui/learnstorybook.com/pull/506/commits/1a10fc896969cd8e84c3d3d6b8f6d22f358075c8) commit so no worries about adding them again

**Comment by merodiro** [comment]
I finished my PR. feel free to review it https://github.com/eihabkhan/learnstorybook.com/pull/1

**Review by eihabkhan — COMMENTED** [review]
Grammar correction reviewed. Thank you @merodiro for the great effort.

File: gatsby-config.js [hunk 1]
```diff
@@ -41,7 +42,8 @@ module.exports = {
           fr: 5.3,
           pt: 5.3,
         },
-        angular: {
+        angul
+        : {
```
Comment (jonniebigodes) [inline gatsby-config.js:46]: @eihabkhan seems that you deleted something here? It should be "angular" not "angul". Also, the Taiwanese translation was just updated and the key "'zh-TW': 5.3," you have here is incorrect. If you wouldn't mind fixing this we'd appreciate it.

**Review by eihabkhan — COMMENTED** [review]
- version for `zh-TW` is now `6.2`
- fixed `angular` typo

**Review by eihabkhan — COMMENTED** [review]
Reviewed changes

File: gatsby-config.js (hunk 1 above)
Comment (eihabkhan) [reply gatsby-config.js]: Fixed 👌

**Comment by jonniebigodes** [comment]
@eihabkhan just ran a final check on this and asides from one minor issue, don't worry it's not related to you, but how Gatsby+Markdown+ ltr (left to right) content is shown, we're good to go. Thanks for factoring in the changes proposed by @merodiro. 

And on that topic @merodiro thank you very much for the time you took to proofread this and help @eihabkhan get the translation merged. We really appreciate it 🙌🏼  !

@eihabkhan one final thing I would like to ask of you. If you don't mind popping into our [Discord Server](https://discord.gg/UUt2PJb) and message me (same username) to follow up on one item. 

Sounds good?

Stay safe both of you

**Comment by eihabkhan** [comment]
> @eihabkhan just ran a final check on this and asides from one minor issue, don't worry it's not related to you, but how Gatsby+Markdown+ ltr (left to right) content is shown, we're good to go. Thanks for factoring in the changes proposed by @merodiro.
> 
> And on that topic @merodiro thank you very much for the time you took to proofread this and help @eihabkhan get the translation merged. We really appreciate it 🙌🏼 !
> 
> @eihabkhan one final thing I would like to ask of you. If you don't mind popping into our [Discord Server](https://discord.gg/UUt2PJb) and message me (same username) to follow up on one item.
> 
> Sounds good?
> 
> Stay safe both of you

Sounds great, thank you @jonniebigodes.

**Comment by eihabkhan** [comment]
> @eihabkhan just ran a final check on this and asides from one minor issue, don't worry it's not related to you, but how Gatsby+Markdown+ ltr (left to right) content is shown, we're good to go. Thanks for factoring in the changes proposed by @merodiro.
> 
> And on that topic @merodiro thank you very much for the time you took to proofread this and help @eihabkhan get the translation merged. We really appreciate it 🙌🏼 !
> 
> @eihabkhan one final thing I would like to ask of you. If you don't mind popping into our [Discord Server](https://discord.gg/UUt2PJb) and message me (same username) to follow up on one item.
> 
> Sounds good?
> 
> Stay safe both of you

Sounds great! 
Thank you @jonniebigodes 
Thank you @merodiro

**Pull request merged**